Explorer/Assets/Scripts/Global/Dynamic/MainSceneLoader.cs

```diff
@@ -176,14 +181,25 @@ private async UniTask InitializeFlowAsync(CancellationToken ct)
                     return;
                 }
 
+
                 Entity playerEntity;
                 (globalWorld, playerEntity) = bootstrap.CreateGlobalWorldAndPlayer(bootstrapContainer, staticContainer!, dynamicWorldContainer!, debugUiRoot);
 
                 dynamicWorldContainer!.InitializeWorldRelatedModules(globalWorld.EcsWorld, playerEntity);
                 staticContainer!.PlayerEntityProxy.SetObject(playerEntity);
 
                 await bootstrap.LoadStartingRealmAsync(dynamicWorldContainer!, ct);
+
                 await bootstrap.UserInitializationAsync(dynamicWorldContainer!, globalWorld, playerEntity, splashScreen, ct);
+
+                //TODO: Implement loading this (or more addresses) from Unleash
+                if (debugSettings.portableExperiencesEnsToLoad != null && staticContainer != null)
```

Comment:
I think no need to check `staticContainer != null`, since its used above: `staticContainer!`

Reply:
you are totally right :)